media-libs/libwebm/libwebm-1.0.0.27.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="WebM VP8 Codec SDK"
```

Comment:
Hmm, what do you think of the following variant?
```
SUMMARY="The SDK of the VP8 video codec developed by Google"
```
(This might be better than "WebM File Parser" for a user who has never bumped into the "word" `WebM`, but I'm OK with anything else.)